### ISSUE TITLE
uncaptcha'd status messages issues on status page

### ISSUE THREAD

**Issue by jtp10181** [RocketMap/RocketMap#1805]
## Expected Behavior

I would expect once the account is uncaptcha'd the message in status would change

## Current Behavior

Usually seems to continue saying the account is waiting for a token until it gets back into the active accounts and scans something

## Possible Solution

Fix so the uncaptcha'd message makes it to the status screen

## Steps to Reproduce (for bugs)

1. Run scanner for a while with manual (or auto?) captcha solving
2. Solve some captcha's and check status screen
3. See if status still shows accounts waiting for token that have not started scanning again

## Context

I will provide some debugging info at bottom in a few minutes

## Your Environment

* Version used: Current Pull https://github.com/RocketMap/RocketMap/commit/5d2485e63dd222dc8a29234e1c0a07180c5cd1ff
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Windows 7 x64

**Here is the logging messages**
**My timezone is UTC-6 so I would assume that's what the 6 hour difference is between the two logging sources.**

```
2017-02-04 15:05:53,987 [search-worker-14][       captcha][ WARNING] Account jtpScan10 has encountered a captcha. Waiting for token.
2017-02-04 15:06:22,921 [captcha-solver-53][       captcha][    INFO] Waking up account jtpScan10 to verify captcha token.
2017-02-04 15:06:22,921 [captcha-solver-53][      auth_ptc][    INFO] PTC User Login for: jtpScan10
2017-02-04 15:06:23,421 [captcha-solver-53][       account][   DEBUG] Login for account jtpScan10 successful.
2017-02-04 15:06:44,178 [captcha-solver-53][       captcha][    INFO] Account jtpScan10 successfully uncaptcha'd, returning to active duty.
```

**Here is what it was showing in the status page**

![image](https://cloud.githubusercontent.com/assets/99599/22622006/0ccb7f5c-eaf6-11e6-9041-18285dacf8c2.png)

**Here is the actual mysql data for confirmation**
![image](https://cloud.githubusercontent.com/assets/99599/22622168/148d8fc0-eaf9-11e6-8c1d-d7263b04e460.png)

**I also noticed this uncaptcha status message from the wrong account get saved in scanner 51**

![image](https://cloud.githubusercontent.com/assets/99599/22622057/fac647a0-eaf6-11e6-8c25-c349f5e0ffe3.png)

Here is some logs from the 6 account if that helps
```
2017-02-04 15:06:17,592 [search-worker-10][       captcha][ WARNING] Account XXXXX6 has encountered a captcha. Waiting for token.
2017-02-04 15:06:37,970 [captcha-solver-54][       captcha][    INFO] Waking up account XXXXX6 to verify captcha token.
2017-02-04 15:06:37,970 [captcha-solver-54][      auth_ptc][    INFO] PTC User Login for: XXXXX6 
2017-02-04 15:06:38,484 [captcha-solver-54][       account][   DEBUG] Login for account XXXXX6 successful.
2017-02-04 15:06:59,278 [captcha-solver-54][       captcha][    INFO] Account XXXXX6 successfully uncaptcha'd, returning to active duty.
```

**Comment by neskk** [comment]
@jtp10181 Thanks for your help, indeed there were some errors because we were using `status['username']` instead of `account['username']`. `status` dict is updated by the worker and since the worker after captcha handling would pick up another account, when we checked its status we would be checking the status of another account, not the one that got captcha'd.
Fixed in #1796 ;)

**Comment by sebastienvercammen** [comment]
Fixed in #1796.
